### DISCUSSION TITLE
Set charge current limit

### DISCUSSION THREAD

**Discussion by SylvainGa** [timdorr/tesla-api#473] [Q&A] [answered]
Anybody found the command to limit the charging current that was introduced in 2021.36.5?

**Comment by GaPhi** [comment, marked as answer]
set_charging_amps command with charging_amps as a parameter (in A)

**Comment by SylvainGa** [comment]
Excellent! I'll add this feature to the Garmin Tesla app.

Thanks for the quick response. I whish responses were that fast in the Teslamate github repository :-(

**Comment by GaPhi** [comment]
I just created 2 pull requests for 3 new charging commands. Hopefully it
will be quickly merged here and integrated in Teslamate, TeslaJS & NodeRed
! :-)

**Reply by SylvainGa** [reply]
Nice :-)

**Comment by GaPhi** [comment]
Pull requests created for TeslaJS & NodeRed too... Just have to wait
for the maintainers now.
